api/queries_repo.go

```diff
@@ -221,6 +221,20 @@ func RepoNetwork(client *Client, repos []ghrepo.Interface) (RepoNetworkResult, e
 	return result, nil
 }
 
+func RepoNetworkStubResponse(owner, repo, permission string) string {
```

Comment:
This moves the non-trivial response stub more closer to the actual implementation so it can be kept in sync with RepoNetwork.